rpm/python-cryptography.spec

```diff
@@ -0,0 +1,151 @@
+%bcond_without python3
+%bcond_with python2
+
+%bcond_with tests
+
+%{!?python3_pkgversion:%global python3_pkgversion 3}
+
+%define srcname cryptography
+
+Name:           python-%{srcname}
+Version:        3.3.3
+Release:        1%{?dist}
+Summary:        PyCA's cryptography library
+
+License:        ASL 2.0 or BSD
+URL:            https://cryptography.io/en/latest/
```

Comment:
URL should be https://github.com/sailfishos/python-cryptography here.

Reply:
That's wrong it should point to upstream see:
https://rpm-packaging-guide.github.io/#rpm-packaging-workspace
https://docs.fedoraproject.org/en-US/quick-docs/creating-rpm-packages/#con_rpm-spec-file-overview

Reply:
No, see here: https://sailfishos.org/wiki/Software_Packaging#RPM_packaging_Guidelines "URL should point to our upstream git tree location for open source packages"